packages/story-editor/package.json

```diff
@@ -65,6 +62,7 @@
 		"jest": "^27.0.6",
 		"jest-matcher-deep-close-to": "^2.0.1",
 		"mockdate": "^3.0.5",
-		"stylis-plugin-rtl": "^1.0.0"
-	}
+		"stylis-plugin-rtl": "^1.0.0",
+    "react-dom": "^17.0.2"
```

Comment:
Broken indentation, wrong order, and wrong import in general 🤔  Remove?

Reply:
No, we it still needed for one rest sadly.

Reply:
Whereabouts?

Reply:
here
https://github.com/google/web-stories-wp/blob/83acc384b40cbed3a7fc52e73edf762595cc00fd/packages/story-editor/src/components/library/test/mediaElement.js#L20

Reply:
Ugh we should fix these tests to properly use  React Testing Library  instead.

Using `element.focus()` and things like `fireEvent` where applicable.